### DISCUSSION TITLE
Enforce alias namespace rules for Sarafu.network

### DISCUSSION THREAD

**Discussion by kamikazechaser** [grassrootseconomics/sarafu.network#86] [General]
The sarafu.network address aliasing closely follows the same spec as a[ VPA (Virtual Payment Address)](https://paytm.com/blog/payments/upi/virtual-payment-address-know-how-to-create-vpa-benefits/) used in some countries for cross bank, cross payment app routing.

In short:

alias@namespace

Correct name spacing now may allow us to create a unified alias resolution system in the future that can span sarafu.network forks and implementations.

## Proposal.

Limit it to:

alias@sarafu or alias@ge where alias can be any alphanumeric text.

**Comment by WillRuddick** [comment]
makes sense - we would have to clean up those that already made aliases. 
@sarafu could work

**Comment by williamluke4** [comment]
I prefer @sarafu

**Comment by WillRuddick** [comment]
It is irritating to have to type @sarafu everytime .... perhaps interfaces could make this automatic?
@ge is also easier and even @ge

**Comment by WillRuddick** [comment]
possibly just leaving open ... it's just a username for now.

**Comment by williamluke4** [comment]
@WillRuddick  @kamikazechaser Currently `name@space` is enforced. Could this be kept or should be make it completely open

**Comment by kamikazechaser** [comment]
I think we leave it open for now. Maybe just a character limit.
